app/src/main/res/layout/activity_product_picker.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_margin="16dp"
+    tools:context=".productpicker.ProductPickerActivity">
+
+    <Button
+        android:id="@+id/success"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="Successful Load"
```

Comment:
Can you put these in a strings file? I know it's just demo code but it's trivial to do